README.md

```diff
@@ -5,14 +5,15 @@
 <img src="./images/preview.PNG" alt="Screeenshot taken on 11th October, 2020" align="center">
 </div>
 
```

Comment:
Could you include some language open-source badges here so that the readme looks a bit attractive?